lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -89,33 +109,35 @@ sub version_info {
     if ($@) {
         handle_exception('version_info', $@, '003');
     }
-    
+
     return \%ver;
 }
 
 $json_schemas{profile_names} = joi->object->strict;
 sub profile_names {
-    my ($self) = @_;
+    my ( $self, undef, undef, $config ) = @_;
+    $self->{config} //= $config;
```

Comment:
What's the point of having a config member if we're getting the config through arguments anyway?

Reply:
The config member is required to use the `_init_db` method. You are right this method is not called here, this is a leftover from my numerous copy/paste.
However it is required in other methods calling `_init_db`. This is because there are places where we specify the database to use even if there is one in the .ini (in the t/*.t files for instance).

I think it would be even better to pass the $config to the constructor and not to the dispatch calls.

I'll remove this line here and in `get_language_tags` and `get_data_from_parent_zone` as there is no use for it.

Reply:
I see, that makes sense. We should be able to provide the Z::B::RPCAPI constructor with arguments like so: https://metacpan.org/pod/JSON::RPC#BASIC-USAGE.